app/react-native/readme.md

```diff
@@ -72,6 +72,21 @@ For RN apps:
 
 Once your app is started, changing the selected story in web browser will update the story displayed within your mobile app.
 
+## Using Haul-cli
+
+[Haul](https://github.com/callstack-io/haul) is an alternative to the react-native packager and has several advantages in that it allows you to define your own loaders, and handles symlinks better.
+
+If you want to use haul instead of the react-native packager, modify they storybook npm script too:
```

Comment:
Yeah, I know this pain, for one of my blogs we use [danger-prose](https://github.com/dbgrandi/danger-prose) to lint for typos.